README.md

```diff
@@ -225,11 +225,14 @@ A number of options are available to allow you to customise the SDK:
 
 - **`steps {List} optional`**
 
-  [link-to-steps-to-components-map]:src/components/App/StepComponentMap.js
+  List of the different steps in the flow. Each step is defined by a named string. The available steps are:
```

Comment:
i think the concept of steps is kinda important to the JS SDK dev, but i agree it deserves better explanation.  I will try breaking that into subheadings and add more explanation - actually I may even combine that with the "options" bit underneath somehow